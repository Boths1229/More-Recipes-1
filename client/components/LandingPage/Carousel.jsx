import React from 'react';
import banner from '../../public/images/banner-img-4.jpg';
import bannerTwo from '../../public/images/banner-img-2.jpg';
import bannerOne from '../../public/images/banner-img-1.jpg';

const Carousel = () => (
  <div id="carouselExampleIndicators" className="carousel slide" data-ride="carousel">
    <ol className="carousel-indicators">
      <li data-target="#carouselExampleIndicators" data-slide-to="0" className="" />
      <li data-target="#carouselExampleIndicators" data-slide-to="1" className="active" />
      <li data-target="#carouselExampleIndicators" data-slide-to="2" className="" />
    </ol>
    <div className="carousel-inner">
      <div className="carousel-item">
        <img className="d-block w-100" src={bannerOne} alt="First slide" />
        <div className="carousel-caption d-none d-md-block">
          <h1 className="cal-big">Paunch bunger be thrice the person you wanna be</h1>
        </div>
      </div>
      <div className="carousel-item active">
        <img className="d-block w-100" src={bannerTwo} alt="Second slide" />
        <div className="carousel-caption d-none d-md-block">
          <h1 className="cal-big">Cheese Pizza spicy blue cheese char! </h1>
        </div>
      </div>
      <div className="carousel-item">
        <img className="d-block w-100" src={banner} alt="Third slide" />
        <div className="carousel-caption d-none d-md-block">
          <h1 className="cal-big">Paunch bunger be thrice the person you wanna be</h1>
        </div>
      </div>
    </div>
    <a
      className="carousel-control-prev"
      href="#carouselExampleIndicators"
      role="button"
      data-slide="prev"
    >
      <span className="carousel-control-prev-icon" aria-hidden="true" />
      <span className="sr-only">Previous</span>
    </a>
    <a
      className="carousel-control-next"
      href="#carouselExampleIndicators"
      role="button"
      data-slide="next"
    >
      <span className="carousel-control-next-icon" aria-hidden="true" />
      <span className="sr-only">Next</span>
    </a>
  </div>
);

export default Carousel;
