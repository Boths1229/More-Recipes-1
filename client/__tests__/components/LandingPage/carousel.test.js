import React from 'react';
import { shallow } from 'enzyme';
import Carousel from '../../../components/LandingPage/carousel';


describe('Carousel Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Carousel />);
    expect(wrapper).toMatchSnapshot();
  });
});
