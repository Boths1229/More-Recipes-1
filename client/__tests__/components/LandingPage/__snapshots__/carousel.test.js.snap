// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Carousel Component should render correctly 1`] = `
<div
  className="carousel slide"
  data-ride="carousel"
  id="carouselExampleIndicators"
>
  <ol
    className="carousel-indicators"
  >
    <li
      className=""
      data-slide-to="0"
      data-target="#carouselExampleIndicators"
    />
    <li
      className="active"
      data-slide-to="1"
      data-target="#carouselExampleIndicators"
    />
    <li
      className=""
      data-slide-to="2"
      data-target="#carouselExampleIndicators"
    />
  </ol>
  <div
    className="carousel-inner"
  >
    <div
      className="carousel-item"
    >
      <img
        alt="First slide"
        className="d-block w-100"
        src="test"
      />
      <div
        className="carousel-caption d-none d-md-block"
      >
        <h1
          className="cal-big"
        >
          Paunch bunger be thrice the person you wanna be
        </h1>
      </div>
    </div>
    <div
      className="carousel-item active"
    >
      <img
        alt="Second slide"
        className="d-block w-100"
        src="test"
      />
      <div
        className="carousel-caption d-none d-md-block"
      >
        <h1
          className="cal-big"
        >
          Cheese Pizza spicy blue cheese char! 
        </h1>
      </div>
    </div>
    <div
      className="carousel-item"
    >
      <img
        alt="Third slide"
        className="d-block w-100"
        src="test"
      />
      <div
        className="carousel-caption d-none d-md-block"
      >
        <h1
          className="cal-big"
        >
          Paunch bunger be thrice the person you wanna be
        </h1>
      </div>
    </div>
  </div>
  <a
    className="carousel-control-prev"
    data-slide="prev"
    href="#carouselExampleIndicators"
    role="button"
  >
    <span
      aria-hidden="true"
      className="carousel-control-prev-icon"
    />
    <span
      className="sr-only"
    >
      Previous
    </span>
  </a>
  <a
    className="carousel-control-next"
    data-slide="next"
    href="#carouselExampleIndicators"
    role="button"
  >
    <span
      aria-hidden="true"
      className="carousel-control-next-icon"
    />
    <span
      className="sr-only"
    >
      Next
    </span>
  </a>
</div>
`;
